docs/build/configure/webhooks.md

```diff
@@ -11,35 +11,91 @@ Webhooks allow you to trigger actions when certain types of data are sent from y
 For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
 Viam provides two webhook types depending on the event you want to trigger on:
 
-- `"part_data_ingested"`: trigger when a sensor on the machine part collects a new reading
-- `"part_online"`: trigger when the machine part is online
+- Data has been synced to the cloud: trigger when data from the machine is synced
+- Part is online: trigger continuously at specified interval while the {{< glossary_tooltip term_id="part" text="machine part" >}} is online
+- Part is offline: trigger continuously at specified interval while the machine part is offline
 
 To configure a webhook:
 
-{{< tabs name="Types of Webhooks" >}}
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
 
 1. Go to the **CONFIGURE** tab of your machine on the [Viam app](https://app.viam.com).
-   Select **JSON** mode.
+   Click the **+** (Create) button in the left side menu and select **Webhook**.
+
+   {{<imgproc src="/build/configure/webhook-create.png" resize="x400" declaredimensions=true alt="The Create menu with Webhook at the bottom of the list of options." >}}
+
+2. Name the webhook and click **Create**.
+
+3. Select the type of event to trigger on from the **Type** dropdown.
+
+4. Follow the instructions depending on the type of webhook you want to implement:
+
+{{< tabs name="Types of Webhooks" >}}
+{{% tab name="Data synced to cloud" %}}
```

Comment:
```suggestion
{{% tab name="Data has been synced to the cloud" %}}
```
just to match to avoid confusion

Reply:
I don't love super long tab names since on narrow screens I think it looks bad

Reply:
maybe just data sync and then more explanation can be in the tab?

Reply:
Okay since I added the sentence below I think it's okay as-is